Week-1/IshaanG/spec.py

```diff
@@ -0,0 +1,45 @@
+import requests
```

Comment:
You should consider adding Module/Class/Method and docstring as well.

It's easier for coders to understand what your module/class/method does.